architecture/core_concepts/routes.adoc

```diff
@@ -1052,3 +1053,53 @@ spec:
     in the subdomain `example.com`. `*.example.com` is the subdomain for host
     name `wildcard.example.com` to reach the exposed service.
 ====
+
+[[disable-namespace-ownership-check]]
+== Disabling Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.  This
+feature relaxes that route ownership policy.
+
+For exanple, if namespace ns1 creates the oldest route r1  www.abc.xyz, then it
+owns that host name and subdomain (abc.xyz) for wildcard routes.  if another
+namespace ns2 creates a route with say a different path www.abc.xyz/path1/path2,
+then it would fail because a route in another namespace (ns1 in this case) "owns"
+that host.
+
+It becomes more complex with
+xref:../../install_config/router/default_haproxy_router.html#using-wildcard-routes[wildcard routes]
+as you now also have to look at who owns "abc.xyz" (or basically has a route *.abc.xyz).
+
+
+The disable namespace ownership check relaxes those restrictions and allows
+hosts (and subdomains) to be claimed across namespaces.
+
+For example, with ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK=true, if
+namespace ns1 creates the oldest route r1  www.abc.xyz,  it owns only
+the hostname (+ path).  Any other namespace (e.g. ns2) can now create
+a route r2  www.abc.xyz/p1/p2  and it would be admitted.  Similarly
+another namespace (ns3) can also create a route  wildthing.abc.xyz
+with a subdomain wildcard policy and it can own the wildcard.
```

Comment:
Maybe reword `and it can own the wildcard` to `it will be granted its claim for the wildcard host name`? We basically want to say that another namespace can create a wildcard route even though it doesn't have the oldest route in that subdomain (`abc.xyz`) and we could potentially have others namespaces claiming other non-wildcard overlapping hosts (ala: foo.abc.xyz, bar.abc.xyz, baz.abc.xyz) and their claims would be granted.